libraries/AP_Scripting/applets/VTOL-quicktune.lua

```diff
@@ -0,0 +1,387 @@
+--[[
+ fast tuning of VTOL gains for multirotors and quadplanes
+
+ This should be used in QLOITER mode for quadplanes and LOITER mode
+ for copters, although it will work in other VTOL modes
+
+--]]
+
+local PARAM_TABLE_KEY = 8
+local PARAM_TABLE_PREFIX = "QUIK_"
+
+local is_quadplane = false
+local atc_prefix = "ATC"
+
+-- bind a parameter to a variable
+function bind_param(name)
+   local p = Parameter()
+   assert(p:init(name), string.format('could not find %s parameter', name))
+   return p
+end
+
+-- add a parameter and bind it to a variable
+function bind_add_param(name, idx, default_value)
+   assert(param:add_param(PARAM_TABLE_KEY, idx, name, default_value), string.format('could not add param %s', name))
+   return bind_param(PARAM_TABLE_PREFIX .. name)
+end
+
+-- setup quicktune specific parameters
+assert(param:add_table(PARAM_TABLE_KEY, PARAM_TABLE_PREFIX, 7), 'could not add param table')
+
+local QUIK_ENABLE      = bind_add_param('ENABLE',         1, 0)
+local QUIK_AXES        = bind_add_param('AXES',           2, 7)
+local QUIK_DOUBLE_TIME = bind_add_param('DOUBLE_TIME',    3, 10)
+local QUIK_GAIN_MARGIN = bind_add_param('GAIN_MARGIN',    4, 70)
+local QUIK_OSC_SMAX    = bind_add_param('OSC_SMAX',       5, 5)
+local QUIK_YAW_P_MAX   = bind_add_param('YAW_P_MAX',      6, 0.5)
+local QUIK_YAW_D_MAX   = bind_add_param('YAW_D_MAX',      7, 0.01)
```

Comment:
This is too low. All my quads end up with a yaw d value above this (usually between 0.01 and 0.1)

Reply:
that is why it is a parameter. The defaults are based on advice from Leonard. For your vehicles you'll want it to be higher